libs/blocks/modal/modal.js

```diff
@@ -18,6 +18,8 @@ function findDetails(hash, el) {
 
 function closeModal(modal) {
   const { id } = modal;
+  const closeEvent = new Event('modal:closed');
```

Comment:
The event could be a bit more specific `milo:modal:closed`? I feel like `modal: closed` could also be used by some 3rd party scripts leading to funky behaviour in the future

Reply:
great callout, will change it!